pkg/istio/rbac/common/util.go

```diff
@@ -96,6 +98,24 @@ func (a *ApiHandler) Delete(item *Item) error {
 	return err
 }
 
+// CheckIfMemberIsAllUsers returns namespace for Athenz domain when role member is of form '<athenz-domain>.*'.
+// Example: domain.sub-domain.* -> domain-sub--domain
+func CheckIfMemberIsAllUsers(member *zms.RoleMember, domainName zms.DomainName) (string, error) {
```

Comment:
Could we rename this to `CheckIfMemberIsAllUsersFromDomain` or something along those lines? I think AllUsers is a little bit confusing because we also support `user.*` which allows all athenz users and services access.

Reply:
Agreed, done - https://github.com/yahoo/k8s-athenz-istio-auth/pull/58/commits/db86ddc3af039aebaa3d8d72ec673964855b4ce2.